src/react-components/src/__tests__/components/Heading-test.tsx

```diff
@@ -15,14 +15,34 @@ describe("Section Headings", () => {
   });
   it("Throws error when invalid heading number passed", () => {
     expect(() => Enzyme.mount(<Heading id="h1" level={9} text={"Heading 9"} />))
-    .to.throw("Heading only supports levels 1-6");
+      .to.throw("Heading only supports levels 1-6");
   });
   it("Can pass heading content as child", () => {
     wrapper = Enzyme.shallow(<Heading id="h1" level={1}>Text</Heading>);
     expect(wrapper.find("h1")).to.have.lengthOf(1);
   });
-  it("Throws error when invalid heading is passed as child", () => {
-    expect(() => Enzyme.mount(<Heading id="h1" level={3}><span>oh no</span></Heading>))
-    .to.throw("Headings can only be plain text or bold");
+  it("Can pass heading content as child span", () => {
+    wrapper = Enzyme.shallow(<Heading id="h1" level={1}>Text <span>hi</span></Heading>);
+    expect(wrapper.find("h1")).to.have.lengthOf(1);
+  });
+  it("Throws error when neither child nor text is passed", () => {
```

Comment:
Not necessary for this PR: It'd be nice to organize the tests so that there are two levels for `describe("Section Headings")`. One would be for errors `describe("errors")` and one for expected renders `describe("renders")` (those names could be better but just an example). For larger components, we like to break it down so we also have a level for component behaviors `describe("behaviors")`.